test/e2e/page-objects/pages/dialog/snap-install.ts

```diff
@@ -18,19 +33,23 @@ class SnapInstall {
 
   private readonly approveButton = '[data-testid="confirmation-submit-button"]';
 
-  private readonly connectButton = '[data-testid="confirm-btn"]';
```

Comment:
this selector was used by 3 different buttons, which caused a lot of confusing on debugging which step we are performing. This has been split in the 3 different selectors, by their text, for better readability and debugging